internal/cmp/lb_monitors.go

```diff
@@ -24,31 +24,27 @@ func newLoadBalancerMonitor(loadBalancerClient *client.LoadBalancerAPIService) *
 }
 
 func (lb *loadBalancerMonitor) Read(ctx context.Context, d *utils.Data, meta interface{}) error {
-	var lbMonitorResp models.GetSpecificLBMonitorResp
+	setMeta(meta, lb.lbClient.Client)
+	var lbMonitorResp models.CreateLBMonitorReq
 	if err := tftags.Get(d, &lbMonitorResp); err != nil {
 		return err
 	}
 
-	lbDetails, err := lb.lbClient.GetLoadBalancers(ctx)
```

Comment:
Just for my understanding, why was this here before? Why has it been removed?

Reply:
Before I have not used datasource to pick the loadBalancer details now i have used datasource from there i can able to create its configurations.